app/content/team/index.js

```diff
@@ -71,174 +140,332 @@ function Index() {
+                      style={styles.input}
+                      value={newName}
+                      onChangeText={(text) => setNewName(text)}
+                      placeholder="Update team name"
                     />
-                  </Menu>
-                ) : (
-                  <></>
-                )
-              }
-            />
-            <KeyboardAwareScrollView
-              // allows opening links from search results without closing keyboard first
-              keyboardShouldPersistTaps="handled"
-              showsVerticalScrollIndicator={false}
-              stickyHeaderIndices={[3]}
-              refreshControl={
-                <RefreshInvalidate invalidateKeys={invalidateKeys} />
-              }
-            >
-              <View style={{ alignItems: "center" }}>
-                <Image
-                  uri="https://upload.wikimedia.org/wikipedia/en/thumb/1/1b/Oregon_State_Beavers_logo.svg/1200px-Oregon_State_Beavers_logo.svg.png"
-                  style={{ marginTop: 0, width: 131, height: 75 }}
-                />
-              </View>
-              <View style={{ alignItems: "center" }}>
-                <View
-                  style={{
-                    flexDirection: "row",
-                    alignItems: "baseline",
-                  }}
+
+                    {/* Save Button */}
+                    <TouchableOpacity
+                      style={styles.saveChangesButton}
+                      onPress={handleUpdate}
+                    >
+                      <Text style={styles.saveChangesButtonText}>Update</Text>
+                    </TouchableOpacity>
+                  </BottomSheetScrollView>
+                </BottomSheetWrapper>
+                <KeyboardAwareScrollView
+                  // allows opening links from search results without closing keyboard first
+                  keyboardShouldPersistTaps="handled"
+                  showsVerticalScrollIndicator={false}
+                  stickyHeaderIndices={[3]}
+                  refreshControl={
+                    <RefreshInvalidate invalidateKeys={invalidateKeys} />
+                  }
                 >
-                  <Text style={{ marginTop: 0, fontSize: 30, marginRight: 0 }}>
-                    OSU Golf Team
+                  <View style={{ alignItems: "center" }}>
+                    <Image
+                      uri={currentTeamData.pfp}
+                      style={{ marginTop: 0, width: 131, height: 75 }}
+                    />
+                  </View>
+                  <View style={{ alignItems: "center" }}>
+                    <View
+                      style={{
+                        flexDirection: "row",
+                        alignItems: "baseline",
+                      }}
+                    >
+                      <Text
+                        style={{ marginTop: 0, fontSize: 30, marginRight: 0 }}
+                      >
+                        {currentTeamData.name}
```

Comment:
Do we want to limit how long this could be? when testing the update I put in a team name that was extremely long and it looked like this: 
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/25bb1f72-f477-4a57-8995-faba04aa8033)